pkgs/development/python-modules/skyfield/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, certifi, numpy, sgp4, jplephem
+, pandas, ipython, matplotlib, assay
+}:
+buildPythonPackage rec {
+  pname = "skyfield";
+  version = "1.42";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NEf9Opqdq8IIDmpO+1bZiD3s8mGteObJs/GHxPx2Gs4=";
+  };
+
+  propagatedBuildInputs = [ certifi numpy sgp4 jplephem ];
+
+  checkInputs = [ pandas ipython matplotlib assay ];
```

Comment:
please add pytestCheckHook, as that will error if tests aren't ran.

sometimes the sdist have tests filtered out, so github should be pulled in that case

Reply:
Skyfield doesn't use pytest, it uses `assay`. I've changed checkPhase to use `assay --batch skyfield.tests`, but it's trying to pull down external files.

Should I pull them using `fetchurl`, or just disable the tests?

Reply:
I would just disable the network tests. We can hopefully trust upstream to publish only working releases, we want to run tests to ensure that it's not broken.

Reply:
I looked into that, it doesn't appear assay provides a way to disable them.

But, all the files are included in the Git repository, so I'll just change it to pull from that instead.